sql/core/src/test/scala/org/apache/spark/sql/streaming/StreamTest.scala

```diff
@@ -322,7 +322,6 @@ trait StreamTest extends QueryTest with SharedSQLContext with Timeouts {
               new UncaughtExceptionHandler {
                 override def uncaughtException(t: Thread, e: Throwable): Unit = {
                   streamDeathCause = e
-                  testThread.interrupt()
```

Comment:
Remove this line because the added test reaches here and interrupt it will make `awaitTermination` fails with `InterruptException`. It's fine to remove it since the worse case is just waiting for 10 seconds to fail a test.